doc/source/install/cmake.rst

```diff
@@ -0,0 +1,253 @@
+.. -*- rst -*-
+
+How to build Groonga with CMake
+===============================
+
+This document describes how to build Groonga from source with CMake.
+
+To get more details about installing Groonga from the source with CMake on a specific environment,
+find the document for the specific environment from :doc:`/install`.
+
+Install depended software
+-------------------------
+
+Here is lists of depended software for GNU/Linux, UNIX and Windows.
+
+TODO
```

Comment:
We should check and add all required software.
I migrated what was originally a TODO in `unix_cmake.rst`, so it's still a TODO.

https://github.com/HashidaTKS/groonga/blob/0566aceb53825ac9f964adbd490e8a5babb66ae2/doc/source/contribution/development/build/unix_cmake.rst#L21